src/x86/intel_edison_fab_c.c

```diff
@@ -76,6 +76,9 @@ static int mmap_fd = 0;
 static int mmap_size;
 static unsigned int mmap_count = 0;
 
+// PWM 0% duty workaround state variable
+static int pwm_disabled = 0;
```

Comment:
Oh well, who uses multiple PWM channels at once or needs more than 640K of RAM anyway :smile: 

Seriously though, I apparently missed this use case and focused on a single-channel one. I'll have this updated in the evening today.

As for the state variable - generally I just wanted to keep the current semantics where one can set duty and period as well as enable/disable PWM independently. Simply disabling it on 0.0f and enabling any time we do a `write()` is of course less code and simpler logic, but i think that might take someone by surprise and run some servos when they don't expect that.